BlockchainSdk/WalletManagers/NEAR/Network/NEARNetworkProvider.swift

```diff
@@ -9,15 +9,72 @@
 import Foundation
 import Combine
 
-struct NEARNetworkProvider: HostProvider {
-    var host: String { baseURL.hostOrUnknown }
-
+struct NEARNetworkProvider {
     private let baseURL: URL
+    private let provider: NetworkProvider<NEARTarget>
 
     init(
         baseURL: URL,
         configuration: NetworkProviderConfiguration
     ) {
         self.baseURL = baseURL
+        provider = NetworkProvider<NEARTarget>(configuration: configuration)
+    }
+
+    func getProtocolConfig() -> AnyPublisher<JSONRPCResult<NEARNetworkResult.ProtocolConfig>, Error> {
+        return requestPublisher(for: .protocolConfig)
+    }
+
+    func getGasPrice() -> AnyPublisher<JSONRPCResult<NEARNetworkResult.GasPrice>, Error> {
+        return requestPublisher(for: .gasPrice)
+    }
+
+    func getInfo(
+        accountId: String
+    ) -> AnyPublisher<JSONRPCResult<NEARNetworkResult.AccountInfo>, Error> {
```

Comment:
и в этом файле тоже плиз
```suggestion
    func getInfo(accountId: String) -> AnyPublisher<JSONRPCResult<NEARNetworkResult.AccountInfo>, Error> {
```

Reply:
Поправил